templates/view-request/ip-links.tpl

```diff
@@ -65,6 +65,10 @@
        href="{$baseurl}/redir.php?tool=bullseye&amp;data={$ipaddress}">
         Bullseye
     </a>
+    <a id="IPalyzer-{$index}" class="btn btn-sm btn-outline-secondary visit-tracking" target="_blank"
+       href="https://ipalyzer.com/{$ipaddress}">
```

Comment:
Tools not in WMF production should go through the redir.php script which helps strip things like referrers so we don't leak request number <=> IP address mappings to other tools.